### ISSUE TITLE
ErlType does not support parameterized types

### ISSUE THREAD

**Issue by Revent** [ignatov/intellij-erlang#1001]
in PR #999 
not supported parameterized types like here https://github.com/poroh/ersip/blob/master/src/uri/ersip_transport.erl#L38
@kvakvs

**Comment by kvakvs** [comment]
Is a drop in replacement for the old class, it is far from being complete.
Work continues in https://github.com/ignatov/intellij-erlang/tree/feature/typed-20230108

**Comment by kvakvs** [comment]
Is there a feature which is broken by this new code? I'd like to create a test for that/or manual test and fix it.

**Comment by Revent** [comment]
yes, in this example, the definition of parameterized types does not work https://github.com/poroh/ersip/blob/master/src/uri/ersip_transport.erl#L38

**Comment by kvakvs** [comment]
This seems to not work in 0.11.1162 (the November 29 release) either, so the recent commit did not break it, it never worked
I'll continue with my development, when finished this should improve the situation

**Comment by ignatov** [comment]
Could you please provide a bit more context, I'm out of it, and provide a small example of `parameterized types`.
